UPDATING.md

```diff
@@ -24,6 +24,7 @@ assists people when migrating to a new version.
 
 ## Next
 
+- [24128](https://github.com/apache/superset/pull/24128) The `RLS_BASE_RELATED_FIELD_FILTERS` config parameter has been removed. Now the Tables dropdown will feature the same tables that the user is able to see elsewhere in the application using the standard `DatasourceFilter`, and the Roles dropdown will be filtered using the filter defined in `EXTRA_RELATED_QUERY_FILTERS["role"]`.
```

Comment:
Could you move this to the **Breaking Changes** section?

Reply:
Oof, good catch!